tools/pom.xml

```diff
@@ -41,6 +41,7 @@
         <dependency>
             <groupId>org.scala-lang</groupId>
             <artifactId>scala-library</artifactId>
+            <scope>compile,runtime</scope>
```

Comment:
I think  the `compile` scope is a [superset](https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#Dependency_Scope) of the `runtime` scope. As written, this is equivalent to just `compile`

Reply:
updated